core/store/src/trie/shard_tries.rs

```diff
@@ -382,18 +381,34 @@ impl WrappedTrieChanges {
                 "Resharding changes must never be finalized."
             );
 
-            // Filtering trie keys for user facing RPC reporting.
-            // NOTE: If the trie key is not one of the account specific, it may cause key conflict
-            // when the node tracks multiple shards. See #2563.
-            match &change_with_trie_key.trie_key {
-                TrieKey::Account { .. }
-                | TrieKey::ContractCode { .. }
-                | TrieKey::AccessKey { .. }
-                | TrieKey::ContractData { .. } => {}
-                _ => continue,
+            let storage_key = if cfg!(feature = "serialize_all_state_changes") {
+                match &change_with_trie_key.trie_key {
```

Comment:
can you also mention (either a longer comment or maybe a link to gihub?) when do we plan to enable this feature permanently?)

What's stopping us from enabling it right now? (would we get a lot more data in that column?)

Reply:
Don't know what to right here other than "will enable in the near future".
I don't expect any issues, but to reduce the risk of unknown unknown let me first try it on betanet and on testnet/mainnet.

> a lot more data
Unlikely to be a problem